github-actions/DotNet.GitHubAction/README.md

```diff
@@ -0,0 +1,8 @@
+<!--
```

Comment:
This doesn't look completed.

I'm fine merging for now, but I think this gets onboarded once we merge.